app/Services/ContractService.php

```
@@ -3,94 +3,71 @@
+        return $this->contractModel->getContractNumber();
     }
 
+    /**
+     * Get contract PDF.
+     *
+     * @param string $id   Contract ID
+     * @param string $type Type of PDF (stream, download, attach)
+     *
+     * @return mixed PDF output based on type
+     */
     public function getContractPdf($id, $type = 'stream')
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)